### DISCUSSION TITLE
Lua Error When Opening Window Tab/Menus

### DISCUSSION THREAD

**Discussion by txproot** [SFX-WoW/Masque#223] [Issues] [answered]
### Game Flavor

TBC Classic

### Game Version

2.5.2.40045

### Add-On Version

9.1.0

### Description

When opening the guild tab this lua error came up, seems to happen when opening different windows but is quite random.

### Steps To Reproduce

1. Open windows/menu tabs in game

### Error Message

```
Message: [ADDON_ACTION_FORBIDDEN] AddOn 'Masque' tried to call the protected function 'GuildControlSetRank()'.
Time: Tue Sep 14 10:39:48 2021
Count: 1
Stack: [ADDON_ACTION_FORBIDDEN] AddOn 'Masque' tried to call the protected function 'GuildControlSetRank()'.
[string "@Interface_TBC\FrameXML\FriendsFrame.lua"]:2503: in function `GuildControlPopupFrame_Initialize'
[string "@Interface_TBC\FrameXML\FriendsFrame.lua"]:966: in function `FriendsFrame_OnEvent'
[string "*:OnEvent"]:1: in function <[string "*:OnEvent"]:1>

Locals: <none>
```

**Comment by StormFX** [comment, marked as answer]
Masque doesn't call that function. See if you can trigger the error with only Masque and whatever add-on(s) you're skinning with it enabled.
